src/algokit/cli/tasks/mint.py

```diff
@@ -86,13 +87,81 @@ def _validate_asset_name(context: click.Context, param: click.Parameter, value:
     """
 
     if len(value.encode("utf-8")) <= MAX_ASSET_NAME_BYTE_LENGTH:
+        token_metadata_path = context.params.get("token_metadata_path")
+        if token_metadata_path is not None:
+            token_metadata = TokenMetadata.from_json_file(token_metadata_path, None, None)
+            if token_metadata.name != value:
```

Comment:
If name is not set in the metadata file, won't this always raise a badparameter error?